pyproject.toml

```diff
@@ -84,6 +84,9 @@ ldp = [
 local = [
     "sentence-transformers",
 ]
+openreview = [
+    "openreview-py",
```

Comment:
Can you add `openreview` to the `mypy` disables as well, since it has no `py.typed`